config.go

```diff
@@ -33,7 +33,8 @@ const (
 	defaultRPCMaxClients    = 10
 	defaultRPCMaxWebsockets = 25
 
-	walletDbName = "wallet.db"
+	walletDbName     = "wallet.db"
```

Comment:
While we're at it, can't we export these two variables and then use them in the `cmd/dropwtxmgr` package?

Reply:
Sure. I think the params can be unified in `loader.go`, while there are multiple places defining `walletDbName`, one [here](https://github.com/btcsuite/btcwallet/blob/4c5bc1b15d3c0dc0d070ca019f4653ff95657452/wallet/loader.go#L21), one [here](https://github.com/btcsuite/btcwallet/blob/a6f3888450e4a5927dc5fd0b5c359f8fef17af78/cmd/dropwtxmgr/main.go#L31) and [here](https://github.com/btcsuite/btcwallet/blob/74a7124666b49cd7e67da87c61169f4fb83fbd73/config.go#L36).

Reply:
Nice!